docs/en/trust.rst

```diff
@@ -165,7 +204,7 @@ All the endpoints listed below are defined in the `OIDC-FED`_ specs.
 | historical keys           | **GET**                                      |                                |  Trust Anchor   |
 |                           |                                              |Lists the expired and revoked   |                 |
 |                           |                                              |keys, with the motivation of the|  Intermediate   |
-|                           | .well-known/openid-federation-historical-jwks|revocation.                     |                 |
+|                           | /historical-jwks                             |revocation.                     |                 |
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏If there's no specific reason I would keep the `.well-known` prefix as it's consistent with the metadata endpoint.

```suggestion
|                           | .well-known/historical-jwks                             |revocation.                     |                 |
```

Reply:
unfortunately it is something defined (changed) in the openid federation specs.

Reply:
but I got your point in terms of normalization. unfortunately the endpoint must be defined in the federation_entity metadata and the correct URL must be taken from there, despite its normalization as well-known